apache/trafficcontrol
# Adds the Traffic Monitor health client.

Adds to the trafficcontrol-cache-config package a client that runs on trafficserver caches to monitor the health status
of it's parent caches as reported by Traffic Monitor.  The client will utilize the Traffic Server traffic_ctl tool to mark 
parents UP or DOWN based upon the health status from Traffic Monitor.

## Which Traffic Control components are affected by this PR?

- Traffic Control Cache Config (T3C, formerly ORT)
- Traffic Monitor

## What is the best way to verify this PR?
Run the unit tests and install on a cache running Trafficserver

Install:
- Build and Install the trafficcontrol-cache-config.rpm
- Rename the sample tm-health-client.json at /etc/trafficcontrol-cache-config/tm-health-client.json.sample to the tm-health-client.json
- Edit the tm-health-client.json per the documentation
- run **systemctl daemon-reload**
- run **systemctl enable tm-health-client**
- run **systemctl start tm-health-client**
- Log file is at /var/log/trafficcontrol-cache-config/tm-health-client.log

It should discover the Traffic Monitors for the relevant CDN from the configured Traffic Ops and begin polling them
with no errors and logging parent cache mark downs when Traffic Monitor determines a parent is un-healthy.

## PR submission checklist
- [] This PR has a CHANGELOG.md entry
